linting/license-check-and-add.config.json

```diff
@@ -1,8 +1,17 @@
 {
   "license": "./utils/tooling/headers/StrimziHeader.txt",
-  "ignore": [".**", ".git/**", "./utils/tooling/headers", "**.md", "**.pptx"],
+  "ignore": [
+    ".**",
+    ".git/**",
+    "./utils/headers",
+    "**.md",
+    "**.pptx",
```

Comment:
we have powerpoint slides??

Reply:
We do! For some of the images in the Architecture document. I made them in Powerpoint, and checked in the source file.